dockers/docker-database/Dockerfile.j2

```diff
@@ -1,5 +1,5 @@
 {% from "dockers/dockerfile-macros.j2" import install_debian_packages, install_python_wheels, copy_files %}
-FROM docker-config-engine-buster
+FROM docker-config-engine-bullseye
```

Comment:
Don't forget the change the dependencies in the file rules/docker-database.mk

Reply:
OK